Install fails "package gitlab.com/NebulousLabs/fastrand: unrecognized import path"

### Background

I follow the install instructions to the point and all is fine until I do "go get -d github.com/lightningnetwork/lnd" to which I get:

> package gitlab.com/NebulousLabs/fastrand: unrecognized import path "gitlab.com/NebulousLabs/fastrand" (parse https://gitlab.com/NebulousLabs/fastrand?go-get=1: no go-import meta tags ())
package gitlab.com/NebulousLabs/go-upnp/goupnp: unrecognized import path "gitlab.com/NebulousLabs/go-upnp/goupnp" (parse https://gitlab.com/NebulousLabs/go-upnp/goupnp?go-get=1: no go-import meta tags ())
package gitlab.com/NebulousLabs/go-upnp/goupnp/dcps/internetgateway1: unrecognized import path "gitlab.com/NebulousLabs/go-upnp/goupnp/dcps/internetgateway1" (parse https://gitlab.com/NebulousLabs/go-upnp/goupnp/dcps/internetgateway1?go-get=1: no go-import meta tags ())


### Your environment


* which operating system - Debian 10 behind TOR


### Steps to reproduce
Just following the install instructions

no, it gives

> make install
/bin/sh: 1: go: not found
/bin/sh: 1: go: not found
 Installing lnd and lncli.
GO111MODULE=on go install -v -tags="" -ldflags " -s -w -X github.com/lightningnetwork/lnd/build.Commit=clock/v1.0.0-167-g60446494e9ea90e50a4753453ba6bb3615dcdd1a -X github.com/lightningnetwork/lnd/build.CommitHash=60446494e9ea90e50a4753453ba6bb3615dcdd1a -X github.com/lightningnetwork/lnd/build.GoVersion= -X github.com/lightningnetwork/lnd/build.RawTags=" github.com/lightningnetwork/lnd/cmd/lnd
/bin/sh: 1: go: not found
make: *** [Makefile:126: install] Error 127



and go is installed as per the instructions

--- a/docs/INSTALL.md
+++ b/docs/INSTALL.md
@@ -104,10 +104,24 @@
 With the preliminary steps completed, to install `lnd`, `lncli`, and all
 related dependencies run the following commands:
 ```
-go get -d github.com/lightningnetwork/lnd
-cd $GOPATH/src/github.com/lightningnetwork/lnd
-make && make install
+git clone https://github.com/lightningnetwork/lnd
+cd lnd
+make install
 ```
+
+The command above will install the current _master_ branch of `lnd`. If you
+wish to install a tagged release of `lnd` (as the master branch can at times be
+unstable), then [visit then release page to locate the latest
+release](https://github.com/lightningnetwork/lnd/releases). Assuming the name
+of the release is `v0.x.x`, then you can compile this release from source with
+a small modification to the above command: 
+```
+git clone https://github.com/lightningnetwork/lnd
+cd lnd
+git checkout v0.x.x
+make install
+```
+
 
 **NOTE**: Our instructions still use the `$GOPATH` directory from prior
 versions of Go, but with Go 1.12, it's now possible for `lnd` to live

